app-admin/flannel-wrapper/files/flannel.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=flannel - Network fabric for containers (System Application Container)
+Documentation=https://github.com/coreos/flannel
+After=etcd.service etcd2.service etcd-member.service
+Before=docker.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10
+LimitNOFILE=40000
+LimitNPROC=1048576
+TimeoutStartSec=60
+ExecStartPre=/sbin/modprobe ip_tables
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/lib/coreos/flannel-wrapper1.uuid
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/lib/coreos/flannel-wrapper2.uuid
+ExecStartPre=/usr/bin/mkdir -p /var/lib/coreos
+Environment=FLANNEL_TAG=0.5.5
```

Comment:
It's a tag rather than a version. I can include a version but it does not have to follow the Gentoo versioning scheme. I suggest to keep it simple and bump the revision when the TAG is changed.